workspace/src/main/resources/public/ts/delegates/actions/create.ts

```diff
@@ -118,6 +119,9 @@ export function ActionCreateDelegate($scope: CreateDelegateScope) {
             return isSharedTree;
         }
     }
+    $scope.isSharedTree = function () {
```

Comment:
De la même façon, on peut typer le type de retour.
```suggestion
    $scope.isSharedTree = function (): boolean {
```